src/QMCWaveFunctions/Fermion/MultiDiracDeterminant.h

```diff
@@ -287,8 +313,10 @@ class MultiDiracDeterminant : public WaveFunctionComponent
                                     dotProducts(i5, a1), dotProducts(i5, a2), dotProducts(i5, a3), dotProducts(i5, a4),
                                     dotProducts(i5, a5));
     }
-    default:
-      return DetCalculator.evaluate(dotProducts, it, n);
+    default: {
+      APP_ABORT("Excitation higher than 5 are not implemented. Please contact developers");
```

Comment:
Revere this breaking change now. Let us put a better solution on the caller side.